src/amo/components/HomeCarousel/index.js

```diff
@@ -0,0 +1,87 @@
+/* @flow */
+import React from 'react';
+import { compose } from 'redux';
+
+import { ADDON_TYPE_EXTENSION } from 'core/constants';
+import translate from 'core/i18n/translate';
+import { visibleAddonType } from 'core/utils';
+import Carousel from 'ui/components/Carousel';
+import CarouselSection from 'ui/components/CarouselSection';
+
+import './styles.scss';
+
+
+type PropTypes = {|
+  i18n: Object,
+|};
+
+export class HomeCarouselBase extends React.Component {
+  props: PropTypes;
+
+  render() {
+    const { i18n } = this.props;
+
+    return (
+      <div className="HomeCarousel">
+        <Carousel
+          sections={[
+            (
+              <CarouselSection
+                key="featured-extensions"
+                linkTo={`/${visibleAddonType(ADDON_TYPE_EXTENSION)}/featured/`}
+                styleName="Home-featured-extensions"
+              >
+                <h3>{i18n.gettext('Featured extensions')}</h3>
+
+                <p>{i18n.gettext('Excellent extensions for all situations')}</p>
+              </CarouselSection>
+            ),
+            (
+              <CarouselSection
+                key="youtube-high-definition"
+                linkTo="/addon/youtube-high-definition/"
+                styleName="Home-youtube-high-definition"
+              >
+                <h3>{i18n.gettext('YouTube High Definition')}</h3>
```

Comment:
Now that we're rendering these like this we could probably have a better API for `CarouselSection` items (title/text/etc.). Let me know if you'd rather us do that here or just file an issue to tweak it as-needed. I didn't want to overengineer it especially while we're filling it all in manually for now.

Reply:
Yet, maybe extracting the `CarouselSection` into a new method would improve readability:

```
  getCarouselSections() {
    // return array
  }

  render() {
    // ...
    <Carousel sections={this.getCarouselSections()} />
  }
```